client/src/utils/HTTPClient.js

```diff
@@ -0,0 +1,19 @@
+const Client = {
+  request: async (endpoint, method = 'GET', body) => {
+    const resp = await fetch(endpoint, {
+      method,
+      headers: {
+        'Content-type': 'application/json',
+        body: JSON.stringify(body),
+      },
+    });
+    if (resp.status !== 200) {
```

Comment:
I haven't tested it yet, but you'll probably also want to check for status 201, I would imagine feathers returns a 201 status on successful account creation (Or at least, if feathers is sensible you'd hope it would)